### ISSUE TITLE
Add popular models

### ISSUE THREAD

**Issue by james77777778** [james77777778/keras-image-models#2]
Reference:
- https://huggingface.co/timm?sort_models=downloads#models
- https://huggingface.co/collections/timm/timm-takes-on-the-classics-655ce63dcaa21067cd7d3f4c

Classics:
- [x] densenet
- [x] inceptionv3
- [ ] inceptionresnetv2
- [x] resnet
- [ ] squeezenet
- [x] vgg
- [x] xception

Lightweight
- [ ] edgenext
- [x] efficientnetlite
- [ ] efficientvit
- [x] ghostnet
- [x] ghostnetv2
- [x] ghostnetv3
- [x] lcnet
- [ ] levit
- [x] mobilenetv2
- [x] mobilenetv3
- [x] mobileone
- [x] mobilevit
- [x] mobilevitv2
- [ ] repghostnet
- [x] tinynet

SOTA:
- [x] convmixer
- [x] convnext
- [x] efficientnet
- [x] efficientnetv2
- [ ] eva02
- [x] hgnet
- [x] hgnetv2
- [x] inceptionnext
- [ ] maxvit
- [x] regnet
- [x] repvgg
- [ ] swin transformer
- [x] vit

**Comment by james77777778** [comment]
![image](https://github.com/james77777778/keras-image-models/assets/20734616/4833dd70-ca62-45b7-9819-20fccbac4a80)

**Comment by cjohn001** [comment]
Please add MobilenetV4 to the zoo, seems its now also available in timm. Great project you are building here!
https://blog.roboflow.com/how-to-use-mobilenetv4-for-classification/

**Comment by james77777778** [comment]
> Please add MobilenetV4 to the zoo, seems its now also available in timm. Great project you are building here! https://blog.roboflow.com/how-to-use-mobilenetv4-for-classification/

Yeah, this is on my radar. I would like to implement it, but I currently don't have the bandwidth.
